### DISCUSSION TITLE
Wie kann ich gekaufte ESP-Wroom-32 anschließen, weil die PINs anders belegt sind?

### DISCUSSION THREAD

**Discussion by Tenker1** [tbnobody/OpenDTU#597] [General]
Liebe OpenDTU Freunde,

leider habe ich eine andere ESP-Wroom-32 bei Amazon
https://www.amazon.de/dp/B071JR9WS9
gekauft und die Pins sind anderes belegt.
In dem Schaltplan braucht man die PINs:
G4 = ist der PIN D4 richtig?
G5 = ist der PIN D5 richtig?
**G16 = finde auf meinem Board nicht!**
G18 = ist der PIN D18 richtig?
G19 = ist der PIN D19 richtig?
G23 = ist der PIN D23 richtig?

Habe ich sie richtig verstanden, dass ich alles über den 
**Flash with ESP_Flasher - web version**
flashen kann?
Also ohne die vorherigen Punkte?

Kann jemand so nett sein und mir helfen?

Danke und VG
Tenker

**Comment by Hans-JR** [comment]
Moin, der GPIO16 ist bei diesem Board unter der Bezeichnung "RX2" versteckt. Ja, die Bezeichnungen Gxx entsprechen hier den Pins Dxx.
Und ja, ich habs vor langer Zeit auch über den ESP-Flasher gemacht. Inzwischen natürlich nur noch per OTA.

**Comment by Leonhard67105** [comment]
Schau mal das Pinout an...vielleicht hilft es 

[ESP-S2 Pinout](
![](https://microcontrollerslab.com/wp-content/uploads/2019/02/ESP32-pinout-mapping.png)
(url)

**Comment by Tenker1** [comment]
Herzlichen Dank für Eure schnelle Hilfe.

Ich habe alles verbunden und geflasht und hoffe erfolgreich, weil ich alles einrichten konnte.
Leider bekomme ich keine Daten von meinem HM-1500.
Die Ser.Nr. habe ich mehrmals geprüft, leider im Browser alles ROOT.
Über Einstellungen - Info - System = Chip Status = GRÜN verbunden / Chip Typ nRF24L01+
Meine Fritz DECT zeigt z.Z. 27 W.
**Kann das OpenDTU System bei der niedrigen Leistung funktionieren und Daten vom WR empfangen, der ca. 10m entfern ist?
Was soll ich noch nachprüfen?**

Herzlichen Dan für Eure Hilfe und ein schönes Wochenende
Tenker

P.S. Kann ich irgendwie Bilder meiner Einstellungen hier einfügen?

**Reply by Hans-JR** [reply]
Also, ich betreibe 2 HM300. Einer ist ca. 2m und der Zweite ca. 7m entfernt. Beide haben sich heute Morgen um ca. 8:00 Uhr eingeschaltet (vorher geht natürlich nichts...).
Für die Stromversorgung nutze ich ein 1,5A USB-Netzteil. Die 5V Leitung führe ich mit 2 Windungen durch eine Ferritperle durch. An dem NRF-Modul habe ich einen 100µf Kondensator angeschlossen. Das System läuft seit ca. 4 Wochen. Es gibt pro Tag so etwa 10 Meldungen in der Ereignisanzeige, dass "DTU command failed" .
Also, erstmal näher an den WR gehen. Dann die "Troubleshooting" Tipps durchgehen. Was sagt "Info > NTP"? Der Status sollte auf "synchronisiert" (grün) stehen.
Bilder kannst Du einfach in den Textrahmen reinziehen.

**Comment by Tenker1** [comment]
Hallo Hans-JR,

und danke für Deine schnelle Antwort.
Mein WR und die 4 PV Paneele sind im Garten, ca. 10 m vom  WZ, wo mein OpenDTU (NRF-Modul mit einer Antenne) am Fenster liegt.
NTP Informationen - Synchronisiert 
Sonst musste ich ein Verlängerungskabel nehmen um näher am WR zu sein...
Jetzt regnet bei uns leider...

Danke und VG
Tenker

![Opera Momentaufnahme_2023-02-18_114731_192 168 0 7](https://user-images.githubusercontent.com/125667856/219856300-ec882514-44c4-4250-8556-bd5082cde5d9.png)
![Opera Momentaufnahme_2023-02-18_114925_192 168 0 7](https://user-images.githubusercontent.com/125667856/219856377-e4e41b39-3dc9-4410-b855-d50faf1e2ad4.png)

**Reply by Hans-JR** [reply]
Na da hat er ja schon mal was gefunden. Sieht für mich nach einem "Entfernungsproblem" aus. Hast Du schon mal mit der "Sendeleistung" gespielt? Ich habe meine allerdings auf "Niedrig" stehen.

**Comment by Tenker1** [comment]
Ich habe meine schon auf "Maximum" eingestellt.

Wenn der Regen aufhört, werde ich den OpenDTU direkt an den WR bringen und testen...

**Comment by Tenker1** [comment]
Ist das bei Netzwerk Informationen richtig, dass der AP nicht aktiv ist?

![Opera Momentaufnahme_2023-02-18_134246_192 168 0 7](https://user-images.githubusercontent.com/125667856/219866476-1ffb88fa-54a2-4e53-922a-120c00f02b3b.png)

**Comment by Solarteur** [comment]
Der AP ist beim Einrichten aktiv, damit SSID und Passwort eingeben werden können. Danach ist er nicht aktiv - ist bei mir auch so.

**Comment by laserdoktor** [comment]
Guten Morgen allerseits,
Ich bin diese Woche von AhoyDTU zu OpenDTU gewechselt (mit ESP8266). Ich kam mit dem MQTT Topic nicht zurecht, warum auch immer.
Mit dem ESP32 hah das alles auf Anhieb funktioniert. Die Daten landen jetzt auch im Grafana-Dashboard.
Mir ist aufgefallen, daß des ESP32 gefühlt deutlich wärmer ist als bei Ahoy mit ESP8266. Ist der ESP32 mehr beschäftigt?, Womit auch immer.
Viele Grüße und herzlichen Dank an alle Macher

**Comment by Tenker1** [comment]
Danke Euch, jetzt funzt, aber ich müsste meinen nRF24L01+ ca. 1 m neben dem WR befestigen.

Ist mein Modul defekt oder ein einfacher Chip nRF24L01 auf nRF24L01+ ungelabelt ?

Das ist doch nicht normal, dass ich nur eine Verbindung zum WR bekomme, wenn die Antenne fast direkt am WR befestigt ist!!!

![1676811128345](https://user-images.githubusercontent.com/125667856/219949738-1f676eee-8427-4d73-be98-af8ee61a1efc.jpg)

**Reply by Solarteur** [reply]
Nein, das ist bezüglich der Reichweite nicht normal. Die "Qualität/Empfangs- bzw. Sendeleistung" der nRF24L01+-Module streut sehr stark - ein Austausch wirkt oft Wunder.

Der Bericht hier
https://github.com/tbnobody/OpenDTU/discussions/414
deckt sich mit meinen Erfahrungen. Auch ich habe gut Erfahrungen mit den Modulen von EBYTE gemacht.

**Comment by Leonhard67105** [comment]
Stimme ich zu! Meine open/ahoydtu ist mindestens 20m mit 2 Mauern dazwischen entfernt. Und ich hb 100 % Empfang Von meinem iPhone gesendetAm 19.02.2023 um 14:21 schrieb Solarteur ***@***.***>:﻿
Nein, das ist bezüglich der Reichweite nicht normal. Die "Qualität/Empfangs- bzw. Sendeleistung" der nRF24L01+-Module streut sehr stark - ein Austausch wirkt oft Wunder.
Der Bericht hier
#414
deckt sich mit meinen Erfahrungen. Auch ich habe gut Erfahrungen mit den Modulen von EBYTE gemacht.

**Comment by Tenker1** [comment]
Guten Morgen an alle,

gestern habe ich von Amazon den "NRF24L01+ von EBYTE" (6 €) bekommen, angeschlossen und alle Verbindungsprobleme waren sofort weg.
Herzlichen Dank Euch allen, für schnelle Hilfe.

Liebe Grüße
Tenker
